### PR TITLE
Security: update copy for 2FA page

### LINKED ISSUE #22606: Security: Revise copy explaining 2FA status
When reviewing the [new 2FA instructions](https://github.com/Automattic/wp-calypso/pull/22573#event-1481309767) added by @simison to [wordpress.com/me/security/two-step](https://wordpress.com/me/security/two-step), I noted that the existing copy describing 2FA is somewhat longer, "colder," and more passive than it could be. 

Per @simison's suggestion, I'm proposing new copy here:

**Existing copy** (minus "Please" in the second paragraph):
36379789-96c641ca-1588-11e8-8c03-f39fca243be6

**Suggested copy**:
> You've enabled Two-Step Authentication on your account -- smart move! When you log in to WordPress.com, you'll need to enter your username and password, as well as a unique passcode generated by an app on your mobile device.

>Switching to a new device? [Follow these steps](https://en.support.wordpress.com/security/two-step-authentication/#moving-to-a-new-device) to avoid losing access to your account.

MIkael, please note that I introduced another small tweak to the second paragraph, namely switching "being locked out of your account," which is a tad passive/clunky to "losing access to your account."

cc @designsimply, in case you have any thoughts / feedback on the proposed edits.

### PR DESCRIPTION
**Pull request by simison**
Closes #22606, follow up for #22573 

Addresses **Me → Security → 2FA** copy being somewhat longer, "colder," and more passive than it could be: https://wordpress.com/me/security/two-step

New text by @benhuberman 

Before:
screen shot 2018-02-23 at 11 00 34

After:
screen shot 2018-02-23 at 11 00 25

## Test

Check http://calypso.localhost:3000/me/security/two-step still works and looks good.

### CODE REVIEW

**Comment by simison** [comment]
There's also SMS 2FA, which has this text currently:

> Your account is currently protected by Two-Step Authentication. While enabled, logging in to WordPress.com requires you to enter a unique passcode, sent via text message, in addition to your username and password.
> 
> Authentication codes are currently being sent to <strong>`SMS-NUMBER`</strong>. If you wish to change this number, please disable Two-Step Authentication, then go through the setup wizard again.


When disabling 2FA:

> You are about to disable Two-Step Authentication. This means we will no longer ask for your authentication code when you sign into your `USERLOGIN` account.
>
> This will also disable your Application Passwords, though they will be available again if you choose to re-enable Two-Step Authentication in the future. If you do choose to re-enable it, please note that you will need to generate new backup codes.
>
> To verify that you wish to disable Two-Step Authentication, please enter the verification code from your device or a backup code and click "Disable Two-Step."

Lemme know if you'd like to update these, too!

**Comment by benhuberman** [comment]
@simison -- for the first message, I'd switch the first paragraph ("Your account is currently protected...") with the new copy ("You've enabled Two-Step Authentication..."). For the second paragraph, if the idea is that this happens when a user changes devices, I'd suggest:

> You're all set to receive authentication codes at `SMS-NUMBER`. Want to switch to a different number? No problem! You'll need to disable Two-Step Authentication, then complete the setup process again on another device.

For the disabling 2FA message, here are some suggested tweaks:

> You are about to disable Two-Step Authentication. This means we will no longer ask for your authentication code when you sign in to your account as `USERLOGIN`.

> This will also disable your Application Passwords, though you can access them again if you ever re-enable Two-Step Authentication. If you decide to re-enable Two-Step Authentication, keep in mind you'll need to generate new backup codes.

> To verify that you wish to disable Two-Step Authentication, enter the verification code from your device or a backup code, and click "Disable Two-Step."

File: client/me/security-2fa-disable/index.jsx [hunk 1]
```diff
@@ -141,17 +143,17 @@ class Security2faDisable extends Component {
...
+								"Two-Step Authentication, keep in mind you'll need to " +
+								'generate new backup codes.'
 						) }
 					</p>
 					<p>
 						{ translate(
-							'To verify that you wish to disable Two-Step ' +
-								'Authentication, please enter the verification code from your ' +
-								'device or a backup code and click "Disable Two-Step."'
+							'To verify that you wish to disable Two-Step Authentication, ' +
```
Comment (a8ci18n) [inline client/me/security-2fa-disable/index.jsx]: Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'To verify that you wish to disable Two-Step Authentication, please enter the verification code from your device or a backup code and click "Disable Two-Step."' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/me-security-2fa-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by simison** [comment]
Updated the PR with new texts by @benhuberman. I also added `<strong>` tag around the username when disabling SMS 2FA.

image

File: client/me/security-2fa-disable/index.jsx (hunk 1 above)
Comment (simison) [reply client/me/security-2fa-disable/index.jsx]: @benhuberman how do we typically do with these? The difference between the old one vs new one is `please` and one `,`.

Old:
> To verify that you wish to disable Two-Step Authentication, please enter the verification code from your device or a backup code and click "Disable Two-Step."

New:
> To verify that you wish to disable Two-Step Authentication, enter the verification code from your device or a backup code, and click "Disable Two-Step."

**Comment by enejb** [comment]
The code looks good. 👍

**Pull request merged**